.github/workflows/release.yml

```diff
@@ -94,6 +92,15 @@ jobs:
         if: ${{ env.TAG == 'latest' && env.FEATURE_COUNT == '0' }}
         run: echo "RELEASE_KIND=patch" >> $GITHUB_ENV
 
+      - name: Run installation
+        run: npm ci
+
+      - name: Test
+        run: npm test
+        env:
+          FORCE_COLOR: 1
+          INFURA_KEY: ${{ secrets.TEST_INFURA_KEY }}
+
```

Comment:
What was the reason for moving these?

Reply:
Peace of mind! The test step takes a few minutes, and everything after it moves pretty quick. It's nice to be able to confirm the output of the previous steps (such as the `TAG` and that the release is correctly registered as pre-release). We've now confirmed that the pre-release stuff works for alpha, so it's a bit less of a concern, but we've still never done a beta or rc release so it will be nice for when we run those for the first time.